### PR TITLE
Issue#21595 fix

### PR DESCRIPTION
**Pull request by DumDereDum**
Linked issue: #21595

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under Apache 2 License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [ ] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: modules/imgproc/test/test_histograms.cpp [hunk 1]
```diff
@@ -1993,6 +1993,41 @@ TEST(Imgproc_Hist_Calc, badarg)
     EXPECT_NO_THROW(cv::calcBackProject(&img, 1, channels, hist, backProj, NULL, 1, true));
 }
 
+#ifdef HAVE_IPP
+TEST(Imgproc_Hist_Calc_IPP, ranges_with_equal_exponent)
```
[the remark this answers is not in the document]
Comment (DumDereDum) [reply modules/imgproc/test/test_histograms.cpp]: done

**Comment by DumDereDum** [comment]
@alalek 
I will create new branch from 3.4 and create new PR

**Comment by alalek** [comment]
Rebasing does NOT require re-opening PRs.

Posted 100+ times this instruction:


This patch should go into [3.4 branch](https://github.com/opencv/opencv/wiki/Branches) first.
We will merge changes from 3.4 into 4.x regularly (weekly/bi-weekly).

Please:
- change "base" branch of this PR: 4.x => 3.4 (use "Edit" button near PR title)
- rebase your commits from 4.x onto 3.4 branch. For example:
    `git rebase -i --onto upstream/3.4 upstream/4.x`
    (check list of your commits, save and quit (Esc + "wq" + Enter)
    where `upstream` is configured by following [this GitHub guide](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and fetched (`git fetch upstream`).
- push rebased commits into source branch of your fork (with `--force` option)

Note: no needs to re-open PR, apply changes "inplace".

**Comment by DumDereDum** [comment]
@alalek sorry next time I'll do this way
but I have just re-open #21635

**Pull request closed without merging**